src/Command/FindAndUploadFilesCommand.php

```diff
@@ -187,40 +187,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 'Failed to get current working directory, command might be searching through unexpected directories and files'
             );
         }
-
         /** @var string $baseDirectory */
         $baseDirectory = $input->getArgument(self::ARGUMENT_BASE_DIRECTORY);
-
-        $io->writeln('Getting supported dependency file names from Debricked', OutputInterface::VERBOSITY_VERBOSE);
-
-        try {
-            $dependencyFileNamesResponse = $api->makeApiCall(
-                Request::METHOD_GET,
-                '/api/1.0/open/files/supported-formats'
-            );
-            $dependencyFileFormats = \json_decode($dependencyFileNamesResponse->getContent(), true);
-        } catch (TransportExceptionInterface $e) {
-            $io->error("Failed to get supported dependency file names: {$e->getMessage()}");
-
-            return 1;
-        } catch (ClientExceptionInterface|RedirectionExceptionInterface|ServerExceptionInterface $e) {
-            /* @noinspection PhpUnhandledExceptionInspection */
-            $io->error("Failed to get supported dependency file names: {$e->getResponse()->getContent(false)}");
-
-            return 1;
-        }
-
-        $directoriesToExcludeString = \strval($input->getOption(self::OPTION_DIRECTORIES_TO_EXCLUDE));
         $searchDirectory = "{$workingDirectory}/{$baseDirectory}";
         $searchDirectory = preg_replace('#/+#', '/', $searchDirectory); // remove duplicate slashes.
 
-        $finder = new Finder();
-        $finder->ignoreDotFiles(false);
-        $finder->files()->in($searchDirectory);
-        if (empty($directoriesToExcludeString) === false && \is_array(
-                $directoriesToExcludeArray = \explode(',', $directoriesToExcludeString)
-            )) {
-            $finder->exclude($directoriesToExcludeArray);
+        $directoriesToExcludeString = \strval($input->getOption(self::OPTION_DIRECTORIES_TO_EXCLUDE));
+        $directoriesToExcludeArray = [];
+        if (empty($directoriesToExcludeString) === false) {
+            $directoriesToExcludeArray = \explode(',', $directoriesToExcludeString) ?? [];
```

Comment:
Extract logic in private method with a descriptive name

Reply:
I was considering it. What made me not do it was the fact that this section is heavily connected to the rest, and thus I deemed it to be more readable this way.